code/style.css

```diff
@@ -1,7 +1,367 @@
+  top: 4%;
+  left: 3%;
+  width: 320px;
+}
+
+.tab {
+  display: none;
+}
+
+.tab-header {
+  width: 48%;
+  text-align: center;
+  display: inline-block;
+  font-size: 18px;
+  cursor: pointer;
+}
+
+.tab-header:after {
+  display: block;
+  content: "";
+  height: 20px;
+  border-bottom: 2px solid rgb(173, 56, 120);
+  transform: scaleX(0);
+  transition: transform 250ms ease-in-out;
+}
+
+.tab:checked + label:after {
+  transform: scaleX(1);
+}
+
+.header {
+  margin: 10px 0 10px 0;
+  font-size: 20px;
+  text-align: center;
+  color: rgba(255, 255, 255, 0.8);
+}
+
+.form-input input {
+  border-radius: 10px;
+  height: 35px;
+  margin: 10px 0;
+  width: 100%;
+  border-width: 0;
+  background-color: rgba(255, 255, 255, 0.2);
+  padding: 10px;
+  color: #fff;
+}
+
+input::placeholder,
+#check {
+  color: rgba(255, 255, 255, 0.8);
+}
+
+input:focus {
+  outline: none;
+}
+
+#check {
+  font-family: "PT Sans", sans-serif;
+  font-size: 2rem;
+}
+
+.submit-button {
+  margin: 20px 0;
+  width: 100%;
+  height: 35px;
+  border-radius: 10px;
+  border-width: 0;
+  text-align: center;
+  background-color: rgb(173, 56, 120);
+  color: #fff;
+}
+
+.form-submit {
+  margin: 20px 0;
+  width: 100%;
+  height: 35px;
+  border-radius: 10px;
+  border-width: 0;
+  text-align: center;
+  background-color: rgb(173, 56, 120);
+  color: #fff;
+}
+
+/* ACCORDION */
+
+main h1 {
+  margin: 2rem 0;
+  font-size: 2.5rem;
+  padding: 20px;
+}
+
+.accordion {
+  max-width: 300px;
+  margin: 5rem 5rem 5rem;
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.accordion-item {
+  color: #111;
+  margin: 1rem 0;
+  border-radius: 0.5rem;
+  box-shadow: 0 2px 5px 0 rgba(0, 0, 0, 0.25);
+}
+
+.accordion-item-header {
+  padding: 0.5rem 3rem 0.5rem 1rem;
+  min-height: 3.5rem;
+  line-height: 1.25rem;
+  font-weight: bold;
+  display: flex;
+  align-items: center;
+  position: relative;
+  cursor: pointer;
+}
+
+.accordion-item-header::after {
+  content: "\002B";
+  font-size: 2rem;
+  position: absolute;
+  right: 1rem;
+}
+
+.accordion-item-header.active::after {
+  content: "\2212";
+}
+
+.accordion-item-body {
+  max-height: 0;
+  overflow: hidden;
+  transition: max-height 0.2s ease-out;
+}
+
+.accordion-item-body-content {
+  padding: 1rem;
+  line-height: 1.5rem;
+  border-top: 1px solid;
+  border: 1px solid rgb(173, 56, 120);
+}
+
+footer {
+  display: flex;
+  justify-content: center;
+  margin-top: 30px;
+  background-color: yellow;
+}
+
+footer a {
+  text-decoration: none;
+  font-weight: bolder;
+  height: 10px;
+  line-height: 5px;
+  color: black;
+}
+
+/* MEDIA QUERIES */
+
+/* MOBILE */
+@media screen and (min-width: 768px) {
+  #hero .caption {
+    font-size: 1rem;
+  }
+
+  #hero .caption span {
+    font-size: 1rem;
+  }
+}
+
+@media screen and (max-width: 768px) {
+  #hero .caption {
+    font-size: 1rem;
+    justify-content: last baseline;
+  }
+
+  #hero .caption span {
+    font-size: 2rem;
+  }
+
+  #hero video {
+    display: none;
+  }
```

Comment:
So the questions is, what do you wanna display instead? Because right now there's no header just a plain grey background instead of an image or video so please check on this to display a responsive header 👌

Reply:
I wanted to display the video but during a lecture we were told "no videos on mobile", so I chose not to display it. Added a background image to the hero-id now, to be displayed instead of  the video.